src/client/inputhandler.cpp

```diff
@@ -58,7 +58,10 @@ void KeyCache::populate()
 	key[KeyType::SCREENSHOT] = getKeySetting("keymap_screenshot");
 	key[KeyType::TOGGLE_HUD] = getKeySetting("keymap_toggle_hud");
 	key[KeyType::TOGGLE_CHAT] = getKeySetting("keymap_toggle_chat");
-	key[KeyType::TOGGLE_FORCE_FOG_OFF] = getKeySetting("keymap_toggle_force_fog_off");
+	key[KeyType::TOGGLE_FOG] = getKeySetting("keymap_toggle_fog");
+	if(key[KeyType::TOGGLE_FOG] == "KEY_F3" && g_settings->exists("keymap_toggle_force_fog_off")){
+		key[KeyType::TOGGLE_FOG] = getKeySetting("keymap_toggle_force_fog_off");
+	}
```

Comment:
Let's not bother with compatibility, it's a rarely used or rebound key and this is 0.5.

Reply:
ok